s3/tests/unit.tftest.hcl

```
@@ -223,6 +223,45 @@ run "aws_s3_bucket_lifecycle_configuration_no_prefix_unit_test" {
   }
 }
 
+run "aws_s3_bucket_data_migration_unit_test" {
+  command = plan
+
+  variables {
+    config = {
+      "bucket_name" = "dbt-terraform-test-s3-x-account",
```

For disambiguation...
```suggestion
      "bucket_name" = "dbt-terraform-test-s3-cross-account",
```